PactNet/IPactBuilder.cs

```diff
@@ -9,6 +9,7 @@ public interface IPactBuilder
         IPactBuilder HasPactWith(string providerName);
         IMockProviderService MockService(int port, bool enableSsl = false);
         IMockProviderService MockService(int port, JsonSerializerSettings jsonSerializerSettings, bool enableSsl = false);
+        IMockProviderService MockService(int port, bool rewriteLocalhost, JsonSerializerSettings jsonSerializerSettings, bool enableSsl = false);

```

Comment:
Can we please remove this, and just add an additional optional param to the end of the above method (after enableSsl = false)? Also can we call `rewriteLocalhost` something like `bindOnAllAdapters`?